src/java.base/share/classes/com/sun/crypto/provider/AESCrypt.java

```diff
@@ -676,12 +676,22 @@ private void makeSessionKey(byte[] k) throws InvalidKeyException {
         // and expand them into arrays of ints.
         int[] expandedKe = expandToSubKey(Ke, false); // decrypting==false
         int[] expandedKd = expandToSubKey(Kd, true);  // decrypting==true
-
+        Arrays.fill(tk, 0);
+        for (int[] ia: Ke) {
+            Arrays.fill(ia, 0);
+        }
+        for (int[] ia: Kd) {
+            Arrays.fill(ia, 0);
+        }
         ROUNDS_12 = (ROUNDS>=12);
         ROUNDS_14 = (ROUNDS==14);
         limit = ROUNDS*4;
 
         // store the expanded sub keys into 'sessionK'
```

Comment:
Update the comment with "erase the previous values in sessionK or other similar wording.

Reply:
OK.